staging/src/k8s.io/kubectl/pkg/cmd/delete/delete.go

```diff
@@ -133,7 +133,7 @@ type DeleteOptions struct {
 	Result        *resource.Result
 
 	genericclioptions.IOStreams
-	warningPrinter *printers.WarningPrinter
```

Comment:
looks like 5 commands added this private field to their helpers... for consistency, we should probably make them all public

Reply:
Fixed.